File: backend/JobPortalWebsite/JobPortal.API/Models/Job.cs
namespace JobPortal.API.Models
{
    public class Job
    {
        public int Id { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public string? Location { get; set; }
        public string? Company { get; set; }
        public string? Salary { get; set; }
        public string? Type { get; set; }
        public int PostedBy { get; set; }
    }
}

File: frontend/src/app/models/job.ts
export interface Job {
  id: number;
  title: string | null;
  description: string | null;
  location: string | null;
  company: string | null;
  salary: string | null;
  type: string | null;
  postedBy: number;
}

export interface CreateJobRequest {
  title: string | null;
  description: string | null;
  location: string | null;
  company: string | null;
  salary: string | null;
  type: string | null;
}
